Topics/Tech_Stacks/ReactNative.md

```diff
@@ -1,30 +1,51 @@
-1. React Native
+# React Native
+## Developing a Mobile App with Expo
+
+React Native is a convenient JavaScript framework created by Meta Platforms Inc. for developing applications. Specifically, its ability to natively render mobile applications on both iOS and Andriod platforms. In addition to its cross-platform functionality, it is primarily based off of React, allowing programmers to immigrate to this framework at ease. Furthermore, React Native is an open-source framework that consists of numerous libraries to assist your application development. <br>
+
+Before my CSC301 advanture, I had no basics in React nor React Native. However, React Native is not a tough framework to learn, so don't stress out and feel free to dive into the [basic documentations](https://reactnative.dev/docs/tutorial) and [beginner courses](https://www.codecademy.com/learn/learn-react-native)! Many [interactive examples](https://reactnative.dev/docs/getting-started) are provided by the developers for you to test and play around with.
+
+Meanwhile, [EXPO](https://docs.expo.dev) is a open-source platform built around React Native. It contains a series of useful tools for developing a React Native app. This includes convenient commands to create and test your app universally on iOS, Android and web without additional configuration. Hence, it is highly recommended to use Expo if you are beginning the development of a mobile app. Furthermore, even existing React Native apps can have the `expo` package installed. <br>
+
+With Expo's [EAS Update](https://docs.expo.dev/eas-update/introduction/) and [EAS Build](https://docs.expo.dev/build/introduction/) features, development builds can be launched instantly, and your app can be tested on mobile devices or mobile device simulators. Official builds can also be launched efficiently to share among your team, or even make its way to AppStore and Google Play. Additionally, Expo offers a [web dashboard](https://expo.dev) with concise UI that aids the management of projects and their deployment. You can sign up for a free Expo account and manage your projects on this dashboard. The [Expo-CLI](https://docs.expo.dev/more/expo-cli/) is the command-line tool that connects all these tools together, and even helps to securely manage your project's dependencies.<br>
 
```

Comment:
SunZhen expanded on the explanation of React Native, including more comprehensive resources for beginners learning React Native. Additionally, he introduced Expo, highlighting its importance as a tool for mobile device testing.

Reply:
Thanks!